odoo/addons/base/models/ir_model.py

```diff
@@ -1993,7 +1981,7 @@ def name_get(self):
 
     # NEW V8 API
     @api.model
-    @tools.ormcache('xmlid')
+    @tools.ormcache('xmlid', cache='xmlid')
```

Comment:
seems really specific for a cache of xmlid. could be in the same than templates probably, but again... right name ? 

I see more the multi cache like:
default = short cache (like before ormcache())
templates = middle cache
assets/routing = long cache

And is it correct to only invalidate a cache without invalidate shorter cache :thinking: ? We should maybe have a 'priority' (10 = short, 100=long) and invalidate all smaller.
I will be not surprised that update xml-id could impact some cached view into the template cache or even cached result from the default cache... based on xmlid ... we could miss them... 

EDIT: to discuss with @odony since it seems not going with his opinion...  <https://github.com/odoo/odoo/pull/119813/files#r1254242740>

Reply:
The current cache in prod don't invalidating the all caches when the long term one is invalidated
They have actually very different invalidation cause. The way it is splited now look to make sense IMHO.

And an xmlid can surprisingly be modified in multiple flows, we don't want to invalidate everything in this case. 
It is not 100% useful to have a dedicated cache, but without that the test_related_field will needed to be adapted.

Reply:
> in prod don't invalidating the all caches

yes it is true, but because we have 3 caches without link. Routing / Assets and the rest.
Here, now that we have multiples small caches, we will have more links between them.


> And an xmlid can surprisingly be modified in multiple flows

I will be curious to know/see the use case. Except during installation... I don't see in practice somes good cases where we update it frequently... :thinking:

Reply:
The main one I remember was the call to _update_xmlids chart_templates.